bindings/js/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.wast
```

Comment:
Hum, it is still useful because when you run `make dist/wasm/package.json` it copies all the npm package material from `bindings/js` into `dist/wasm` ; if you don't keep the `.gitignore` then your git tree is going to be dirty with things in `dist/wasm` after your `make` call.

Reply:
Aaah you're right about the `bindings/js/.gitignore`, I'll delete it. I thought you were talking about `dist/wasm/.gitignore`.